sdk/attestation/azure-security-attestation/src/samples/java/com/azure/security/attestation/AttestationClientJavaDocCodeSnippets.java

```diff
@@ -245,8 +248,8 @@ public static void attestOpenEnclaveAsync1() {
         // END: com.azure.security.attestation.AttestationAsyncClient.getOpenIdMetadata
 
         // BEGIN: com.azure.security.attestation.AttestationAsyncClient.getAttestationSigners
-        Mono<List<AttestationSigner>> signers = client.listAttestationSigners();
-        signers.block().forEach(cert -> {
```

Comment:
>block [](http://example.com/codeflow?start=16&length=5)

Thank you for updating this. Not sure, how I missed it in the last PR.

Reply:
Yeah - I was running through the samples and noticed that the async samples had .block in them - SLOPPY!